packages/mui-base/src/Modal/Modal.tsx

```diff
@@ -91,7 +90,7 @@ const Modal = React.forwardRef(function Modal<RootComponentType extends React.El
     slots = {},
     ...other
   } = props;
-
+  const manager = managerProp as any;
```

Comment:
This is temporary. I think there is some type error in this file. For example,

we should change
```jsx
  const modal = React.useRef<{
    modalRef?: typeof modalRef.current;
    mountNode?: typeof mountNodeRef.current;
  }>({});
```

to

```jsx
  const modal = React.useRef<{
    mount?: Element;
    modalRef?: Element;
  }>({});
```

to match the type of methods provided by `ModalManager`. I'd rather do these changes in a separate PR.

Reply:
Weird. But yes, I agree, let's solve this in a separate PR. Could you just add a TODO comment above this line explaining what is it for?

Reply:
done